core/23LCV1024_1.fzp

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<module moduleId="23LCV1024_24357530b44412a899aa27521bf5766d_1" referenceFile="generic_ic_dip_8_300mil.fzp">
+<module moduleId="23LCV1024_24357530b44412a899aa27521bf5766d_1" fritzingVersion="0.12.34" referenceFile="generic_ic_dip_8_300mil.fzp">
```

Comment:
Do we have some automated way of updating this number? What exactly does this number represent?

Reply:
Mainly in modelbase.cpp:120 ff. Parts without version are treated as parts < 0.4.2 . The code is a bit tricky, because the greater than has its operands in inverse order, plus inverse of its return value is used. So, having no version has the same effect as having a part that was designed pre 0.4.2. I choose 0.12.34 as replacement as hint that the version is set as a fix.